core/src/main/kotlin/com/malinskiy/marathon/analytics/external/graphite/GraphiteTracker.kt

```diff
@@ -0,0 +1,51 @@
+package com.malinskiy.marathon.analytics.external.graphite
+
+import com.malinskiy.marathon.analytics.internal.sub.TestEvent
+import com.malinskiy.marathon.analytics.internal.sub.TrackerInternalAdapter
+import com.malinskiy.marathon.device.DeviceInfo
+import com.malinskiy.marathon.execution.TestStatus
+import com.malinskiy.marathon.test.Test
+import com.malinskiy.marathon.test.toSafeTestName
+
+class GraphiteTracker(
+    private val graphite: GraphiteClient
+) : TrackerInternalAdapter() {
+
+    override fun trackTest(event: TestEvent) {
```

Comment:
I tried using tags but they didn't work for me. It turned out that my company disabled them for some technical reason. I'm not able to implement tags right now, because I can't test them. Also, I wouldn't be able to use the Graphite integration if it used tags. I'd rather not change this now and perhaps add tags later as an option.

Reply:
They're disabled probably because of high cardinality